helm/grafana/templates/organizations/giantswarm-org.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.organizations.giantswarm.enabled }}
+apiVersion: observability.giantswarm.io/v1alpha1
+kind: GrafanaOrganization
+metadata:
+  name: {{ .Values.organizations.giantswarm.name }}
+spec:
+  displayName: {{ .Values.organizations.giantswarm.displayName }}
+  rbac:
+    admins:
+    {{- if $.Values.oidc.giantswarm.write_all_group }}
```

You should check for these strange `$.Values` all around your code.

```suggestion
    {{- if .Values.oidc.giantswarm.write_all_group }}
```

I'm surprised this syntax works, but even then we should have a similar syntax for the whole chart.